repository/project.go

```diff
@@ -139,14 +139,7 @@ func (r *ProjectRepository) UpdateProject(ctx context.Context, projectID uuid.UU
 		return svcmodel.Project{}, err
 	}
 
-	_, err = tx.Exec(ctx, query.UpdateProject, pgx.NamedArgs{
-		"id":                        p.ID,
-		"name":                      p.Name,
-		"slackChannelID":            p.SlackChannelID,
-		"releaseNotificationConfig": model.ReleaseNotificationConfig(p.ReleaseNotificationConfig), // converted to the struct with json tags (the field is saved as json in the database)
-		"githubRepositoryURL":       p.GithubRepositoryURL.String(),
-		"updatedAt":                 p.UpdatedAt,
-	})
+	_, err = tx.Exec(ctx, query.UpdateProject, model.ToUpdateProjectArgs(p))
```

Comment:
Preparing arguments for this query felt like too many lines to have directly in Repository function, therefore I added `model.ToUpdateProjectArgs`